cram_3d_world/cl_bullet_vis/tests/bullet-world-gl-context-test.lisp

```diff
@@ -26,7 +26,7 @@
 ;;; ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 ;;; POSSIBILITY OF SUCH DAMAGE.
 
-(in-package :bt-vis-test)
+(in-package :cl-bullet-vis-tests)
```

Comment:
Hope you tested this part well. To ensure that nobody else uses those names? Please do grep to make sure, and if anyone does use the name, you can always add a nickname.

Reply:
This is only the test-package, the real one is another. I checked it and it's fine. Would be pretty bad if someone used any implementation from a test package. Also, the package's `:nickname` is still the same, and I adapted the package names to their systems, not the system names, so `depends-on` in *.asd should still work without worries.